static/app/views/detectors/components/detectorListRow.tsx

```diff
@@ -59,15 +55,30 @@ export function DetectorListRow({
   );
 }
 
-const StyledCheckbox = styled(Checkbox)<{checked?: boolean}>`
-  visibility: ${p => (p.checked ? 'visible' : 'hidden')};
-  align-self: flex-start;
-  opacity: 1;
-`;
-
-const StyledGraphCell = styled(EmptyCell)`
-  width: 35%;
-`;
+export function DetectorListRowSkeleton() {
+  return (
+    <RowWrapper>
+      <CellWrapper>
+        <div style={{width: '100%'}}>
```

Comment:
I don't normally use `style`, but I figured it was okay here since it's just a skeleton. I can make them styled components too if wanted